vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,10 +4,17 @@ sidebarDepth: 2
 # Build and Publish a Simple Bundle
 
 ## Overview
-This tutorial describes how to create a simple Entando Bundle and deploy it into the [Entando Component Repository](../../../docs/getting-started/concepts-overview.md#entando-component-repository) (ECR). At minimum, this requires manually defining a bundle with a single widget, checking the artifacts into Git, applying the custom resource to Kubernetes, and then installing the bundle into an application.
 
+This tutorial describes how to create a simple Entando Bundle and deploy it into the [Entando Component Repository](../../../docs/getting-started/concepts-overview.md#entando-component-repository) (ECR). This process consists of 4 essential steps:
+
+1. Define the bundle component(s)
+2. Check the bundle artifacts into a Git repository
+3. Apply the bundle custom resource to Kubernetes
+4. Install the bundle into an application
+
+The sections below detail the steps required to generate simple examples of each Entando bundle type.
```

Comment:
what bundle type? may be good to supply idea about the types

detail the steps >> slightly awkward

Reply:
this is a left over fragment from copy/paste; i'd started integrating v5 into this page before we changed course.